srcpkgs/vscode/patches/fix_dir.patch

```diff
@@ -42,5 +42,5 @@ index b975e1094a2c..a094cc36e810 100644
  [Desktop Action new-empty-window]
  Name=New Empty Window
 -Exec=@@EXEC@@ --no-sandbox --new-window %F
-+Exec=/usr/lib/@@NAME@@/@@NAME@@ --no-sandbox --new-window %F
++Exec=/usr/lib/bin/@@NAME@@/@@NAME@@ --no-sandbox --new-window %F
```

Comment:
Should probably be /usr/lib/@@NAME@@/bin/@@NAME@@

Reply:
Well, ok. That you are correct on :) There is a reason the title is `[really wip]` :D